README.md

```diff
@@ -6,37 +6,95 @@
   <img src="https://utils.flavours.dev/presskit/spring_boot_flavour_glossy.svg" width="100">
 </p>
 
-## Project Outline
-This project template utilizes
+A [Flavours](https://www.flavours.dev) project template for Java/Spring Boot
 
-* Java 11
-* [FlywayDB](https://flywaydb.org/) for database migrations;
-* [Gradle](https://gradle.org/) to build and, in dev-mode, run the application with hot reload;
-* [Spring JPA](https://docs.spring.io/spring-data/jpa/docs/current/reference/html/) for database access;
-* [Spring Web](https://spring.io/guides/gs/serving-web-content/) to serve HTTP requests;
-* [Thymeleaf](https://www.thymeleaf.org/doc/tutorials/3.0/thymeleafspring.html) for HTML-templating;
 
-## Local development
-### Quick start
-Run `docker-compose up web` to start a dev-server with hot reloadling enabled.
+## Quick start
+
+### Clone the repository
+
+```
+git clone git@github.com:flavours/getting-started-with-spring-boot.git
+```
+
+### Build the project
 
-### Info
-The container runs a Tomcat server listening on port 8080. The docker-compose.yml file is set up to
-expose this port to the docker host at port 8000, but you are free to change it as you wish.
+```
+cd getting-started-with-spring-boot
+docker-compose build
+```
+
+The project includes a ``web`` service, running the Java code, and a ``db`` service, running a Postgres database.
+See the ``docker-compose.yml`` file for details.
+
+### Run the project
+
+```
+docker-compose up
+````
 
+Containers for both services will be launched. The project can be reached at http://localhost:8000.
 
+Hot-reloading is enabled (i.e. changes to the Java code in the project will cause the application to restart so that they 
+can be used.)
 
-## Getting-started project development
 
-If you want to contribute to this project, please checkout github for our [contribution guide](https://github.com/flavours/getting-started-with-spring-boot/blob/master/CONTRIBUTING.md) and [code of conduct](https://github.com/flavours/getting-started-with-spring-boot/blob/master/CODE_OF_CONDUCT.md).
+## How to
 
-### How to prepare a new release of the getting-started project
+### Run the local project on a different port
 
-To create a new release of the getting started project, you have to updated the default welcome screen with the latest version. 
+The container runs a Tomcat server listening on port 8080. The ``docker-compose.yml`` file is set up to
+expose this port to the Docker host at port 8000, but you are free to change it as you wish - edit the ``ports`` directive:
```

Comment:
this might break because i think the CLI expects 8000 locally

Reply:
our CLI? or some project specific thing?